docs/guides/getting-started-with-vm-operator.md

```diff
@@ -0,0 +1,346 @@
+      volumeClaimTemplate:
+        spec:
+          resources:
+            requests:
+              storage: 2Gi
+    resources:
+      limits:
+        cpu: "0.3"
+        memory: "300Mi"
+  vminsert:
+    replicaCount: 2
+EOF
+```
+</div>
+
+
+The expected output:
+
+```bash
+vmcluster.operator.victoriametrics.com/example-vmcluster-persistent created
+```
+
+* By applying this CRD we install [VictoriaMetrics cluster](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html) to default [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) inside your cluster.
+* `retentionPeriod: "4"` defines [retention](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#retention) to 4 month.
+* `replicationFactor: 2` replication factor for the ingested data, i.e. how many copies should be made among distinct `-storageNode` instances. If the replication factor is greater than one, the deduplication must be enabled on the remote storage side.
+* `replicaCount: 2` creates two replicas of vmselect, vminsert and vmstorage.
+* `storageDataPath: "/vm-data"` will create volume for `vmstorage` at `/vm-data` folder.
+* `resources: ` configures resources for pod. See [k8s resource configuration docs](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers) to get more infomation.
+
+
+Note, that it may take some time for pods to start. To check that pods are started run the following command:
+<div class="with-copy" markdown="1" id="services">
+
+```bash
+kubectl get pods | grep vmcluster
+```
+</div>
+
+The expected output:
+```bash
+NAME                                                     READY   STATUS    RESTARTS   AGE
+vminsert-example-vmcluster-persistent-845849cb84-9vb6f   1/1     Running   0          5m15s
+vminsert-example-vmcluster-persistent-845849cb84-r7mmk   1/1     Running   0          5m15s
+vmoperator-victoria-metrics-operator-67cff44cd6-s47n6    1/1     Running   0          7m16s
+vmselect-example-vmcluster-persistent-0                  1/1     Running   0          5m21s
+vmselect-example-vmcluster-persistent-1                  1/1     Running   0          5m21s
+vmstorage-example-vmcluster-persistent-0                 1/1     Running   0          5m25s
+vmstorage-example-vmcluster-persistent-1                 1/1     Running   0          5m25s
+```
+
+There is an extra command to get information about cluster state:
+<div class="with-copy" markdown="1" id="services">
+
+```bash
+kubectl get vmclusters
+```
+</div>
+
+The expected output:
+```bash
+NAME                           INSERT COUNT   STORAGE COUNT   SELECT COUNT   AGE     STATUS
+example-vmcluster-persistent   2              2               2              5m53s   operational
+```
+
+To scrape metrics from Kubernetes with a VictoriaMetrics Cluster we will need to install [vmagent](https://docs.victoriametrics.com/vmagent.html) with some additional configurations. To do so, please run the following command:
```

Comment:
worth to explain what are these configurations. Specifically, that we're looking for vminsert URL so vmagent can write data into it. 
This is not just for text, this is to give user an idea of what we're doing except just copying commands blindly.